addons/point_of_sale/static/src/customer_display/customer_display_adapter.js

```diff
@@ -17,35 +17,35 @@ export class CustomerDisplayPosAdapter {
     }
 
     dispatch(pos) {
-        if (pos.config.customer_display_type === "local") {
-            this.channel.postMessage(JSON.parse(JSON.stringify(this.data)));
-        }
-
-        if (pos.config.customer_display_type === "remote") {
-            pos.data.call("pos.config", "update_customer_display", [
-                [pos.config.id],
-                this.data,
-                pos.config.access_token,
-            ]);
-        }
-
-        const proxyIP = pos.getDisplayDeviceIP();
-        if (proxyIP) {
-            fetch(`${deduceUrl(proxyIP)}/hw_proxy/customer_facing_display`, {
-                method: "POST",
-                headers: {
-                    Accept: "application/json",
-                    "Content-Type": "application/json",
-                },
-                body: JSON.stringify({
-                    params: {
-                        action: "set",
-                        data: this.data,
+        if (pos.config.is_customer_display) {
```

Comment:
```suggestion
        if (!pos.config.is_customer_display) { return }
```


[Why You Shouldn’t Nest Your Code](https://www.youtube.com/watch?v=CFRhGnuXG-4)